samples/csharp_dotnetcore/11.qnamaker/Controllers/BotController.cs

```diff
@@ -15,21 +17,21 @@ namespace Microsoft.BotBuilderSamples
     [ApiController]
     public class BotController : ControllerBase
     {
-        private readonly IBotFrameworkHttpAdapter _adapter;
-        private readonly IBot _bot;
+        private readonly IBotFrameworkHttpAdapter Adapter;
```

Comment:
Not clear why the naming convention on this changes. _adapter is what I would have expected.